lib/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -89,20 +89,13 @@ public function get_items_permissions_check( $request ) {
 	 * @return WP_Error|WP_REST_Response
 	 */
 	public function get_items( $request ) {
-		$prepared_args = array(
-			'comment__in'     => $request['include'],
-			'comment__not_in' => $request['exclude'],
-			'number'          => $request['per_page'],
-			'post_id'         => $request['post'] ? $request['post'] : '',
-			'parent'          => isset( $request['parent'] ) ? $request['parent'] : '',
-			'search'          => $request['search'],
-			'offset'          => $request['offset'],
-			'orderby'         => $this->normalize_query_param( $request['orderby'] ),
-			'order'           => $request['order'],
-			'status'          => 'approve',
-			'type'            => 'comment',
-			'no_found_rows'   => false,
-		);
+		$prepared_args = WP_REST_Request_Controller::prepare_query_args_from_request( $request );
+		$prepared_args['status'] = 'approve';
+		$prepared_args['type'] = 'comment';
+		$prepared_args['no_found_rows'] = true;
+		$prepared_args['parent'] = ! empty( $prepared_args['parent'] ) ? $prepared_args['parent'] : '';
+		$prepared_args['post_id'] = ! empty( $prepared_args['post_id'] ) ? $prepared_args['post_id'] : '';
```

Comment:
This doesn't actually need to happen, as we're properly setting a default value.